config/foreman.migrations/20231005004305_redis_default_cache.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman'].is_a?(Hash) && answers.dig('foreman', 'rails_cache_store', 'type') == 'file'
```

Comment:
I thought about dig, but didn't suggest it because I wasn't 100% sure if `rails_cache_store` was always a hash or `nil`. Now that I think more about it, I think that's all good.